docs/website/nuxt.config.js

```diff
@@ -49,7 +49,8 @@ export default {
       {
         id: 'UA-141692582-2'
       }
-    ]
+    ],
+    ['@nuxtjs/sitemap', { hostname: 'https://www.talos.dev/' }]
```

Comment:
just curious, what does this buy us?

Reply:
A sitemap is kind of a hint to search engines about the contents of the site. This module auto-generates one based on your routes. (more info: https://support.google.com/webmasters/answer/156184?hl=en)